AntonBabunin/src/artcode/shop/Product.java

```diff
@@ -0,0 +1,51 @@
+package artcode.shop;
+
+public class Product {
+    private String name;
+    private double price;
+    private int id;
+
+    public Product(String name, double price, int id) {
+        this.name = nameProductTester(name);
+        this.price = priceProductTester(price);
+        this.id = idProductTester(id);
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public double getPrice() {
+        return price;
+    }
+
+    public void setPrice(double price) {
+        this.price = price;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    private static String nameProductTester (String name) {
```

Comment:
I think this is not reasonable to keep here static methods with such logic. They may be removed

Reply:
Done.